lib/AppInfo/Application.php

```diff
@@ -88,6 +89,7 @@ public function register(IRegistrationContext $context): void {
 			'getPathForToken',
 			'getWopiForToken',
 		]);
+		$context->registerPreviewProvider(EMF::class, '/image\/emf/');
```

Comment:
Did you give that a test with a Collabora intance setup? We have not migrated our preview class to the IProviderV2 interface (which we really should 🙈 ) and as far as I see it the context registration requires that.

Reply:
> Did you give that a test with a Collabora intance setup?

Yep, it works like a charm ;)

> the context registration requires that.

True, the psalm-param type hints IProviderV2.

I guess it's better to use the deprecated way that expects a IProvider instance for now.

Reply:
Yep, let's stick to that then for now